measures/2019/measures-data.json

```diff
@@ -10869,7 +10869,7 @@
   {
     "category": "ia",
     "title": "Implementation of formal quality improvement methods, practice changes, or other practice improvement processes",
-    "description": "Adopt a formal model for quality improvement and create a culture in which all staff actively participates in improvement activities that could include one or more of the following such as:  -  Multi-Source Feedback; -  Train all staff in quality improvement methods; -  Integrate practice change/quality improvement into staff duties; -  Engage all staff in identifying and testing practices changes; -  Designate regular team meetings to review data and plan improvement cycles; -  Promote transparency and accelerate improvement by sharing practice level and panel level quality of care, patient experience and utilization data with staff; and/or -  Promote transparency and engage patients and families by sharing practice level quality of care, patient experience and utilization data with patients and families, including activities in which clinicians act upon patient experience data.",
+    "description": "Adopt a formal model for quality improvement and create a culture in which all staff actively participates in improvement activities that could include one or more of the following such as:  \r-  Multi-Source Feedback; \r-  Train all staff in quality improvement methods; \r-  Integrate practice change/quality improvement into staff duties; \r-  Engage all staff in identifying and testing practices changes; \r-  Designate regular team meetings to review data and plan improvement cycles; \r-  Promote transparency and accelerate improvement by sharing practice level and panel level quality of care, patient experience and utilization data with staff; and/or \r-  Promote transparency and engage patients and families by sharing practice level quality of care, patient experience and utilization data with patients and families, including activities in which clinicians act upon patient experience data.",
```

Comment:
I saw these "\r-" the last time I changed PI measures, thought is was a weird, it must be coming in from the generator code...

Reply:
It's a Windows linebreak vs. a Unix linebreak (something that has been the bane of my existence as a dev on Windows).